LSP-pyright.sublime-settings

```diff
@@ -10,6 +10,14 @@
 	],
 	// ST4
 	"selector": "source.python",
+    "schemes": [
```

Comment:
Indentation is off :)

Reply:
I will fix this simple one. Let's ship this PR with a `pyright` update.